bokeh/charts/builder/area_builder.py

```diff
@@ -68,55 +69,31 @@ class AreaBuilder(Builder):
         )
         area.show()
     """
-    def __init__(self, values, index=None, stacked=False, legend=False,
-                 palette=None, **kws):
-        """
-        Args:
-            values (iterable): iterable 2d representing the data series
-                values matrix.
-            index (str|1d iterable, optional): can be used to specify a
-                common custom index for all data series as follows:
-                    - As a 1d iterable of any sort that will be used as
-                        series common index
-                    - As a string that corresponds to the key of the
-                        mapping to be used as index (and not as data
-                        series) if area.values is a mapping (like a dict,
-                        an OrderedDict or a pandas DataFrame)
-            stacked (bool, optional): if:
-                True: areas are draw as a stack to show the relationship of
-                    parts to a whole
-                False: areas are layered on the same chart figure. Defaults
-                    to False.
-            legend (str, optional): the legend of your chart. The legend
-                content is inferred from incoming input.It can be
-                ``top_left``, ``top_right``, ``bottom_left``,
-                ``bottom_right``. ``top_right`` is set if you set it
-                 as True. Defaults to None.
-            palette(list, optional): a list containing the colormap as
-                hex values.
-
-        Attributes:
-            source (obj): datasource object for your chart,
-                initialized as a dummy None.
-            x_range (obj): x-associated datarange object for you plot,
-                initialized as a dummy None.
-            y_range (obj): y-associated datarange object for you plot,
-                initialized as a dummy None.
-            groups (list): to be filled with the incoming groups of data.
-                Useful for legend construction.
-            data (dict): to be filled with the incoming data and be passed
-                to the ColumnDataSource in each chart inherited class.
-                Needed for _set_And_get method.
-            attr (list): to be filled with the new attributes created after
-                loading the data dict.
-                Needed for _set_And_get method.
-            index(see inputs): received index input
-        """
-        self.source = None
-        self._stacked = stacked
-        self.index = index
 
-        super(AreaBuilder, self).__init__(values, legend=legend, palette=palette)
+    facet = Bool(False, help="""
```

Comment:
It's being passed in in some of the examples. If the right thing to do is change the examples let's do that now. We should support faceting at some point but that can come later.